src/style.css

```diff
@@ -0,0 +1,24 @@
+html {
+  margin: 0;
+}
+
+.stars .star {
```

Comment:
don't increase specificity
```suggestion
.star {
```